crates/rpc/rpc-eth-types/src/utils.rs

```diff
@@ -1,6 +1,8 @@
 //! Commonly used code snippets
 
+use jsonrpsee::core::RpcResult;
```

Comment:
this can be imported directly from jsonrpsee_core I believe so we don't need the additional dep